src/core_pdf/impl/engine/render/target.py

```
@@ -2023,6 +2111,56 @@ def blit_image(
+        # rules and letter stems of a scanned page. This sits ahead of the
+        # affine dispatch so the rotated path gets an averaged source too.
+        #
+        # Both source axes are held to the *larger* device extent rather than
+        # matched to width and height separately: a quad may rotate the image,
+        # in which case its height runs along the box's width, and reducing per
+        # axis would shrink the wrong one. Taking the maximum can only reduce
+        # less than strictly necessary, never more.
+        scale = self.scale
+        device_extent = max(
+            1,
+            int(math.ceil((box[2] - box[0]) * scale)),
```

Ruff (0.16.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)